src/feature.def

```diff
@@ -22,16 +22,17 @@
  *           variable          flag                       default  help
  * ========================================================================= */
 
-WABT_FEATURE(exceptions,       "exceptions",              false,   "Experimental exception handling")
-WABT_FEATURE(mutable_globals,  "mutable-globals",         true,    "Import/export mutable globals")
-WABT_FEATURE(sat_float_to_int, "saturating-float-to-int", true,    "Saturating float-to-int operators")
-WABT_FEATURE(sign_extension,   "sign-extension",          true,    "Sign-extension operators")
-WABT_FEATURE(simd,             "simd",                    false,   "SIMD support")
-WABT_FEATURE(threads,          "threads",                 false,   "Threading support")
-WABT_FEATURE(multi_value,      "multi-value",             true,    "Multi-value")
-WABT_FEATURE(tail_call,        "tail-call",               false,   "Tail-call support")
-WABT_FEATURE(bulk_memory,      "bulk-memory",             false,   "Bulk-memory operations")
-WABT_FEATURE(reference_types,  "reference-types",         false,   "Reference types (externref)")
-WABT_FEATURE(annotations,      "annotations",             false,   "Custom annotation syntax")
-WABT_FEATURE(gc,               "gc",                      false,   "Garbage collection")
-WABT_FEATURE(memory64,         "memory64",                false,   "64-bit memory")
+WABT_FEATURE(exceptions,          "exceptions",              false,   "Experimental exception handling")
+WABT_FEATURE(mutable_globals,     "mutable-globals",         true,    "Import/export mutable globals")
+WABT_FEATURE(sat_float_to_int,    "saturating-float-to-int", true,    "Saturating float-to-int operators")
+WABT_FEATURE(sign_extension,      "sign-extension",          true,    "Sign-extension operators")
+WABT_FEATURE(simd,                "simd",                    false,   "SIMD support")
+WABT_FEATURE(threads,             "threads",                 false,   "Threading support")
+WABT_FEATURE(function_references, "function-references",     false,   "Typed function references")
```

Comment:
Strange that github can't do better with this diff.  I assume this is just adding the one line and the rest is indentation?

Reply:
Exactly, this is just indentation + one new line